service/src/main/java/uk/gov/hmcts/reform/fpl/enums/SectionType.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+public enum SectionType {
```

Comment:
What value `Type` suffix adds?

Reply:
None, other than consistency with our other enums.

Reply:
In other places it makes more sense IMO.

Type word is used there to highlight it is a type of order or type of party, type of something.

Here I am less reluctant to say type of section. For me it is more section.